### DISCUSSION TITLE
Access host's containers from distrobox

### DISCUSSION THREAD

**Discussion by Joonalai** [89luca89/distrobox#1093] [Q&A] [answered]
Is there a way to access containers running on host from inside distrobox?

I know it is possible to [use docker or podman](https://github.com/89luca89/distrobox/blob/main/docs/useful_tips.md#using-docker-inside-a-distrobox) inside distrobox, but in my use case I am limited in using really old image (Ubuntu 16.04) and with that podman version it is not (at least easily) possible to run podman- or docker-compose there.

**Comment by 89luca89** [comment, marked as answer]
Use `distrobox-host-exec podman` and it will run podman on host
